plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTableBase.java

```diff
@@ -0,0 +1,111 @@
+
+import java.util.Collection;
+import java.util.Map;
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.cubrid.CubridConstants;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.ext.generic.model.GenericUtils;
+import org.jkiss.dbeaver.model.DBPEvaluationContext;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.meta.IPropertyValueListProvider;
+import org.jkiss.dbeaver.model.meta.Property;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+public abstract class CubridTableBase extends GenericTableBase {
+	
+	private CubridUser owner;
+	private CubridObjectContainer container;
+	private String isSystemTable;
+	public CubridTableBase(CubridObjectContainer container, String tableName, String tableType, JDBCResultSet dbResult) {
+		super(container, tableName, tableType, dbResult);
+		this.container = container;
+		String owner_name;
+
+		if (dbResult != null) {
+			owner_name = GenericUtils.safeGetString(container.getCubridTableCache().tableObject, dbResult, CubridConstants.OWNER_NAME);
+			isSystemTable = GenericUtils.safeGetString(container.getCubridTableCache().tableObject, dbResult, CubridConstants.IS_SYSTEM_CLASS);
+		} else {
+			owner_name = getDataSource().getContainer().getConnectionConfiguration().getUserName().toUpperCase();
+			isSystemTable = "";
+		}
+
+		for(CubridUser cbOwner : getUsers()){
+			if(cbOwner.getName().equals(owner_name)) {
+				this.owner = cbOwner;
+				break;
+			}
+		}
+	}
+
+	public boolean isPhysicalTable() {
+		return !isView();
+	}
+
+	public CubridObjectContainer getContainer() {
+		return this.container;
+	}
+
+	public Collection<? extends CubridUser> getUsers() {
+		try {
+			return container.getDataSource().getCubridUsers(null);
+		} catch (DBException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		return null;
+	}
+
+	@Override
+	public String getObjectDefinitionText(DBRProgressMonitor monitor, Map<String, Object> options) throws DBException {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	public Collection<? extends CubridTableIndex> getIndexes(DBRProgressMonitor monitor) throws DBException {
+		if (getDataSource().getInfo().supportsIndexes()) {
+			return this.getContainer().getCubridIndexCache().getObjects(monitor, getContainer(), this);
+		}
+		return null;
+	}
+
+	@Nullable
+	@Property(viewable = true, editable = true, updatable = true, listProvider = OwnerListProvider.class, order = 2)
+	public CubridUser getOwner() {
+		return owner;
+	}
+
+	public void setOwner(CubridUser owner) {
+		this.owner = owner;
+	}
+
+	public String getUniqueName() {
+		return this.owner.getName() + "." + this.getName();
+	}
+
+	@NotNull
+	@Override
+	public String getFullyQualifiedName(DBPEvaluationContext context) {
+		if(isSystemTable.equals("YES")) {
+			return this.getName();

```

Comment:
Please use
DBUtils.getFullQualifiedName(getDataSource(), this);
instead. Because you do not have a quoting condition here.